app/src/main/java/co/techmagic/hr/presentation/time_tracker/TimeTrackerFragment.kt

```diff
@@ -196,6 +230,14 @@ class TimeTrackerFragment : BaseViewFragment<TimeTrackerPresenter>(), TimeTracke
         actionBarChangeListener.setActionBarTitle(title)
     }
 
+    override fun onItemClicked(reportViewModel: UserReportViewModel) {
+        presenter?.onEditTimeReportClicked(reportViewModel)
+    }
+
+    override fun onTrackTimeClicked(position: Int) {
+        TODO("not implemented") //To change body of created functions use File | Settings | File Templates.
```

Comment:
Let's show some toast message instead of crashing app

Reply:
Done